packages/manager/modules/emailpro/src/translations/Messages_fr_FR.json

```
@@ -551,6 +551,7 @@
   "emailpro_tab_INFORMATIONS_summary": "Résumé",
   "emailpro_tab_INFORMATIONS_wait_activation": "En attente d'activation",
   "emailpro_tab_INFORMATIONS_webmail": "Webmail",
+  "emailpro_tab_INFORMATIONS_server": "Server",
```

I would say
```suggestion
  "emailpro_tab_INFORMATIONS_server": "Serveur",
```
but we might want to keep the english translation in this case